CRM/Admin/Page/AJAX.php

```diff
@@ -341,7 +341,7 @@ public static function recipientListing() {
    * Used by jstree to incrementally load tags
    */
   public static function getTagTree() {
-    $parent = CRM_Utils_Type::escape(CRM_Utils_Array::value('parent_id', $_GET, 0), 'Integer');
+    $parent = CRM_Utils_Type::escape(($_GET['parent_id'] ?? 0), 'Integer');
```

Comment:
Before/after should work the same given that it's being passed into `CRM_Utils_Type::escape()`